src/main/java/org/altbeacon/beacon/service/scanner/CycledLeScanner.java

```diff
@@ -157,6 +157,8 @@ public void stop() {
         mScanningEnabled = false;
         if (mScanCyclerStarted) {
             scanLeDevice(false);
+            LogManager.d(TAG, "forcing scanning to stop even for devices capable of multiple detections per cycle");
+            stopScan();
```

Comment:
Why is `stopScan()` required here?

The call to `scanLeDevice(false)` will call it.